src/mapgen/mg_decoration.cpp

```diff
@@ -269,9 +276,42 @@ size_t Decoration::placeDeco(Mapgen *mg, u32 blockseed, v3s16 nmin, v3s16 nmax)
 }
 
 
+void Decoration::cloneTo(Decoration *def) const
```

Comment:
just a question, why implementing clone and not a copy operator directly ?

Reply:
The copy operator was already disabled on these classes, I assume to prevent accidentally copying them. So I implemented it as an explicit function call instead.

Reply:
i think we should find why it's disabled and re-enable it instead of creating our own no ? :)

Reply:
Well I think it's still useful to prevent accidentally copying.